content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -252,7 +252,7 @@ the StatefulSet.
 
 ### Pod index label
 
-{{< feature-state for_k8s_version="v1.28" state="beta" >}}
+{{< feature-state for_k8s_version="v1.28" state="stable" >}}
```

Comment:
Consider using the "feature_gate_name" option for `feature-state `shortcode. This will make it more data-driven by pulling from the feature gate description file.

```suggestion
{{< feature-state feature_gate_name="PodIndexLabel" >}}
```

Reply:
done, PTAL